source/index.html.md

```diff
@@ -132,19 +132,228 @@ curl "https://www.castupload.com/api/v1/actor_profiles/123" \
+        "thumb": "https://d205pfv3qf1itp.cloudfront.net/s3/castupload-com/production/5c5a1832-3ee4-495d-bfb0-03694dab0bd2.png/e63279986f8a8b40f0d1a99bb14c69918a749a34750248578a95ee17fdc58da7/c_fill,f_jpg,g_North,h_190,q_80,w_190"
+      },
+      "orientation": "landscape",
+      "dimensions": [
+        365,
+        360
+      ],
+      "main_picture": false,
+      "year": 2021,
+      "recorded_at": null
+    }
+  ],
+  "external_showreel": "https://www.youtube.com/watch?v=xxxx",
+  "external_showreels": [
+    "https://www.youtube.com/watch?v=xxxx",
+    "https://vimeo.com/xxxx"
+  ],
+  "showreels": [
+    {
+      "name": "Showreel 2021",
+      "url": "https://www.castupload.com/actors/john-doe/showreels/15",
+      "type": "video"
+    },
+    {
+      "name": "Showreel 2020",
+      "url": "https://www.castupload.com/actors/john-doe/showreels/16",
+      "type": "video"
+    },
+    {
+      "name":"Audioreel 2021",
+      "url":"https://www.castupload.com/actors/john-doe/showreels/17",
+      "type":"audio"
+    }
+  ],
+  "vita": {
+    "primary_education": [
+      {
+        "name": "ABC School",
+        "year_from": 2012
+      }
+    ],
+    "award": [
+      {
+        "info": "",
+        "name": "Avanca Film Festival - Best Performance",
+        "year_from": 2017,
+        "year_to": 2017,
+        "award_nominated": true
+      }
+    ],
+    "education": [
+      {
+        "info": "",
+        "name": "123 School",
+        "year_from": 2013,
+        "year_to": 2014
+      }
+    ],
+    "film": [
+      {
+        "info": "",
+        "name": "Schneeflöckchen",
+        "year_from": 2019,
+        "year_to": 2020,
+        "role": "Jack",
+        "role_type": "supporting",
+        "distributor": "",
+        "director": "Adolfo Kolmerer",
+        "producer": "",
+        "caster": ""
+      },
+      {
+        "info": "",
+        "name": "The Harmonica ",
+        "year_from": 2018,
+        "year_to": 2018,
+        "role": "Felix",
+        "role_type": "leading",
+        "distributor": "",
+        "director": "",
+        "producer": "Deutsche Film- und Fernsehakademie Berlin",
+        "caster": ""
+      }
+    ],
+    "television": [
+      {
+        "info": "",
+        "name": "Mord mit Aussicht",
+        "year_from": 2015,
+        "year_to": 2016,
+        "role": "Robert",
+        "role_type": "misc",
+        "distributor": "",
+        "director": "",
+        "producer": "",
+        "caster": ""
+      }
+    ],
+    "theatre": [],
+    "audio": [],
+    "other": [],
+    "internet": [],
+    "commercial": []
+  }
 }
 ```
 
 This endpoint retrieves a specific actor profile.
 
+#### Notes:
+- It is not guaranteed that the example above is complete. Additional fields may be added without notice.
```

Comment:
This may or may not be shown "above" (depending on the device) so let's keep this generic to not confuse unnecessarily.

```suggestion
- It is not guaranteed that the exemplary JSON structure shown is complete. Additional fields may be added without notice.
```